EffPot.m

```diff
@@ -259,6 +280,48 @@
 	];
 
 
+];
+
+(*
+	Calculates the effective potential. (just a copy of CalculatePotentialUS)
+*)
+CalculatePotential[OptionsPattern[]]:=Module[{},
```

Comment:
Should this really be a hard copy?

Reply:
Well they do the exact thing. The user can always define what mode they want beforehand. Optimally I would just rename the CalculatePotential function altogether, but might mess up with some people that already have  CalculatePotentialUS in their scripts.

Reply:
I rather meant that one of the two instances of the function should be derived from the other to not cause copying issues in the future.

Reply:
Ah fair enough. I changed it now